fixtures/parser_tests.js

```diff
@@ -401,6 +447,75 @@ function parserTests(parse: (file: string) => ParseResult, isTypescript = false)
       assertBlock2(itBlock, 2, 7, 6, 9, 'each test %p');
       assertNameInfo(itBlock, 'each test %p', 3, 10, 3, 21);
     });
+
+    it('For tagged template syntax', () => {
```

Comment:
good this one has both test and describe. Same comment as above, let's add some nested `.each` test cases to be thorough

Reply:
I added a separate test fixture for some nested cases.